pkg/agent/nodeportlocal/rules/iptable_rule.go

```diff
@@ -29,6 +28,13 @@ import (
 // NodePortLocalChain is the name of the chain in IPTABLES for Node Port Local
 const NodePortLocalChain = "ANTREA-NODE-PORT-LOCAL"
 
+// PodNodePort contains the Node Port, Pod Port and the Pod IP.
+type PodNodePort struct {
+	NodePort int
+	PodPort  int
+	PodIP    string
+}
+
 // IPTableRules provides a client to perform IPTABLES operations
 type IPTableRules struct {
```

Comment:
Not your code, but this seems another case of internal struct. Probably we should name it iptablesRules.

Reply:
Done.